src/error-handler.js

```diff
@@ -1,3 +1,5 @@
+import FlashMessage, { flash } from 'Components/FlashMessage';
```

Comment:
Not too happy about this to be honest. We are deliberately not including anything in the error handler to make sure that it stays as small as possible and also avoid as many potential points of failure as possible.

And as you said, the UX doesn't seem to be great anyway. Maybe we can think of another way to implement this?